### DISCUSSION TITLE
#57 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by Thegaram** [tog-ETH-er/togETHer#42] [General]
Agenda:

- Meta: calendar invite
- Community updates
- Shanghai meetup
- Plancker DAO activity proposal
- [poap.xyz](https://poap.xyz/) experience and advice
- Fireside discussion

会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会：
https://meet.google.com/wba-mxnt-bsu

**Comment by Thegaram** [comment]
### Crypto Tech Night

- CTN#20: Web3在中国的发展和机遇（23号本周四晚上）。感谢Plancker DAO的支持。欢迎来转发

![image](https://user-images.githubusercontent.com/7571518/220081755-1e4c1cc0-8d38-46cc-b44d-b3ec367e2000.png)

**Comment by 0x-stan** [comment]
# Dapp-Learning

## repo update

5个PR #926 ～ #931

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了2场分享会


- 《比特币NFT的基本原理和特性——Ordinals》by Po
  - https://www.youtube.com/watch?v=RpoDtNwWSBo
 
- 《ChatGPT：A AGI Product》by Suvivor@PlanckerDAO

**Comment by stephanieeee-ho** [comment]
### ECN
- 日常的翻译和 E 群志运营

**Comment by Stamp9** [comment]
## 原语里弄

研究课题进行中。
https://www.shanghai-upgrade.org/ 上海升级系列活动。
https://www.primitiveslane.org/cryptographers 寻找密码朋克专题研究研究员招募中。

**Comment by liushooter** [comment]
## Rebase

- web3 极客日报 14 期 http://daily.rebase.network/
- 1 期 rebase 大学分享： [从 AITEX 看区块链和保险的融合](https://mp.weixin.qq.com/s/EU_Wj-mKaBbrelNdnRWayg)
